app/medInria/medRegistrationWorkspace.cpp

```diff
@@ -11,48 +11,29 @@
 
 =========================================================================*/
 
-#include <medRegistrationWorkspace.h>
-
 #include <dtkCore/dtkSignalBlocker.h>
 
-#include <medViewFactory.h>
-#include <medAbstractView.h>
-
+#include <medAbstractLayeredView.h>
+#include <medLayerParameterGroup.h>
 #include <medRegistrationSelectorToolBox.h>
-#include <medViewContainer.h>
+#include <medRegistrationWorkspace.h>
 #include <medTabbedViewContainers.h>
-#include <medRegistrationSelectorToolBox.h>
-#include <medAbstractLayeredView.h>
-#include <medStringListParameter.h>
-
 #include <medToolBoxFactory.h>
-
+#include <medViewContainer.h>
 #include <medViewParameterGroup.h>
-#include <medLayerParameterGroup.h>
 
 class medRegistrationWorkspacePrivate
 {
 public:
-    medRegistrationSelectorToolBox * selectorToolBox;
-    medViewContainer *fixedContainer;
-    medViewContainer *movingContainer;
-    medViewContainer *fuseContainer;
-
+    QVector<medViewContainer*> containers; // fixed/moving/fuse
     medViewParameterGroup *viewGroup;
-    medLayerParameterGroup *fixedLayerGroup;
-    medLayerParameterGroup *movingLayerGroup;
+    QVector<medLayerParameterGroup*> layerGroups; // fixed/moving
```

Comment:
Putting the containers and layer groups into vectors has resulted in a big loss of code readability (compare ```containers[1]``` with ```fixedContainer```). An enum could improve this.